Mathlib/Order/Monovary.lean

```diff
@@ -310,39 +310,39 @@ end PartialOrder
 variable [LinearOrder ι]
 
 protected theorem Monotone.monovary (hf : Monotone f) (hg : Monotone g) : Monovary f g :=
-  fun i j hij => hf (hg.reflect_lt hij).le
+  fun _ _ hij => hf (hg.reflect_lt hij).le
 #align monotone.monovary Monotone.monovary
 
 protected theorem Monotone.antivary (hf : Monotone f) (hg : Antitone g) : Antivary f g :=
-  (hf.Monovary hg.dual_right).dual_right
+  (hf.monovary (Antitone.dual_right hg)).dual_right
```

Comment:
`hg.dual_right` doesn't work?

Reply:
can you fix
`alias monotone_toDual_comp_iff ↔ _ Antitone.dual_right`
at
https://github.com/leanprover-community/mathlib4/blob/be3b492c47a934ea2fda7307316b6869b91944d8/Mathlib/Order/Monotone/Basic.lean#L244-L244

Reply:
and similar for all other projection notations that are not working?

Reply:
I made a [Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/.60alias.60.20unfolds.20definitions). This file is low priority. Maybe `alias` is easy to fix.